modules/user_accounts/php/edit_user.class.inc

```diff
@@ -304,14 +308,25 @@ class Edit_User extends \NDB_Form
                      'userID'      => $uid,
                     )
                 );
-                $examinerID = $examinerID = $DB->pselectOne(
+                $examinerID = $DB->pselectOne(
```

Comment:
I don't know why we need to extract the new examinerID here, it doesn't seem to be used.

Reply:
@cmadjar I was wrong about this, it is used :+1: